From d4aec311fb5dc7276d463ad486ab009d9aa5e1b5 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Fri, 12 Jul 2019 11:23:32 -0700
Subject: [PATCH 09/21] DO-NOT-UPSTREAM: Disable -Wframe-larger-than= for Clang

There are a few of these warnings remaining but I don't want to see them
at the moment.

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 lib/Kconfig.debug | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index 498d344ea53a..1b77b211b6bf 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -300,6 +300,7 @@ config ENABLE_MUST_CHECK
 config FRAME_WARN
 	int "Warn for stack frames larger than"
 	range 0 8192
+	default 0 if CC_IS_CLANG
 	default 2048 if GCC_PLUGIN_LATENT_ENTROPY
 	default 1280 if (!64BIT && PARISC)
 	default 1024 if (!64BIT && !PARISC)
-- 
2.27.0

